src/student.js

```diff
@@ -0,0 +1,42 @@
+export default class Student {
+  constructor(details) {
+    const { firstName, lastName, gitHubUserName, email, cohortId } = details
+
+    this.firstName = firstName
+    this.lastName = lastName
+    this.gitHubUserName = gitHubUserName
+    this.email = email
+    this.cohortId = cohortId
```

Comment:
never sure on whether linking class instances solely by id isn't a massive headache – because what if I'm silly and delete the cohort lol

Reply:
If you really want to you could guard against deleting a cohort if it has linked students. Personally I don't see it being an issue, if the cohort no longer exists you can reassign the students to a new one with not much issues.